ctl/import_test.go

```diff
@@ -95,18 +86,12 @@ func TestImportCommand_RunValue(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	hldr := test.MustOpenHolder()
-	defer hldr.Close()
-	s := test.NewServer()
-	defer s.Close()
-
-	s.Handler.API.Cluster = test.NewCluster(1)
-	s.Handler.API.Cluster.Nodes[0].URI = s.HostURI()
-	s.Handler.API.Holder = hldr.Holder
-	cm.Host = s.Host()
+	cmd := test.MustRunMainWithCluster(t, 1)[0]
+	hostport := cmd.Server.URI.HostPort()
```

Comment:
this pattern of `hostport` is different from the pattern later in this test where you set `cm.Host` directly and then use that going forward.